Mon 05 Dec 2022 01:47:19 PM EST
Running cachesize experiment. Output goes to plot-cachesize.out
free(): invalid pointer
./run-one-experiment: line 70: 1429178 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429272 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429364 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429456 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429543 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429636 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
free(): invalid pointer
./run-one-experiment: line 70: 1429724 Aborted                 ./server $PORT $NR_THREADS $MAX_REQUESTS $CACHE_SIZE > server.log
Cachesize experiment done.
